### PR TITLE
[docs-only] Change (ocis,OCIS) to oCIS, unify GitHub branding

### PR DESCRIPTION
**Pull request by pascalwengerter**
## Description
Found various cases of `ocis`, `OCIS` and `oCIS` throughout the codebase. `oCIS` seems to be the offical branding so I went ahead and replaced it where applicable (skipping all the parts where it's part of a variable or command, obviously).

Also, changed some instances of `github` to `GitHub`.

## Related Issue
Not applicable

## Motivation and Context
Figured it would be nicer for the project and readers of the documentation to have a uniform branding.

## How Has This Been Tested?
Not necessary

## Types of changes
- [X] Documentation only

### CODE REVIEW

**Comment by phil-davis** [comment]
drone CI failed with:
```
runsh: Total unexpected failed scenarios throughout the test run:
webUISharingFolderAdvancedPermissionMultipleUsers/sharedFolderWithMultipleUsersAdvancedPermissions.feature:59
``
https://drone.owncloud.com/owncloud/ocis/2996/71/7

That is issue #1492 which is currently being worked on.

I restarted drone.

File: tests/acceptance/expected-failures-API-on-OCIS-storage.md [hunk 1]
```diff
@@ -1393,7 +1393,7 @@
 -   [apiShareOperationsToShares/changingFilesShare.feature:59](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiShareOperationsToShares/changingFilesShare.feature#L59)
 -   [apiShareOperationsToShares/changingFilesShare.feature:60](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiShareOperationsToShares/changingFilesShare.feature#L60)
 
-### [OCIS-storage overwriting a file as share receiver, does not create a new file version for the sharer](https://github.com/owncloud/ocis/issues/766)
+### [ocis-storage overwriting a file as share receiver, does not create a new file version for the sharer](https://github.com/owncloud/ocis/issues/766)
```
Comment (phil-davis) [inline tests/acceptance/expected-failures-API-on-OCIS-storage.md:1396]:
```suggestion
### [oCIS-storage overwriting a file as share receiver, does not create a new file version for the sharer](https://github.com/owncloud/ocis/issues/766)
```

File: tests/acceptance/expected-failures-API-on-OCIS-storage.md [hunk 2]
```diff
@@ -2310,7 +2310,7 @@
 -   [apiWebdavUploadTUS/uploadFileMtimeShares.feature:55](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiWebdavUploadTUS/uploadFileMtimeShares.feature#L55)
 -   [apiWebdavUploadTUS/uploadFileMtimeShares.feature:56](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiWebdavUploadTUS/uploadFileMtimeShares.feature#L56)
 
-### [OCIS-storage reading a file that a collaborator uploaded is impossible](https://github.com/owncloud/ocis/issues/763)
+### [ocis-storage reading a file that a collaborator uploaded is impossible](https://github.com/owncloud/ocis/issues/763)
```
Comment (phil-davis) [inline tests/acceptance/expected-failures-API-on-OCIS-storage.md:2313]:
```suggestion
### [oCIS-storage reading a file that a collaborator uploaded is impossible](https://github.com/owncloud/ocis/issues/763)
```

File: tests/acceptance/expected-failures-API-on-OCIS-storage.md [hunk 3]
```diff
@@ -2344,5 +2344,5 @@
 -   [apiTrashbin/trashbinFilesFolders.feature:284](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiTrashbin/trashbinFilesFolders.feature#L284)
 -   [apiTrashbin/trashbinFilesFolders.feature:285](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiTrashbin/trashbinFilesFolders.feature#L285)
 
-### [OCIS-storage overwriting a file as share receiver, does not create a new file version for the sharer](https://github.com/owncloud/ocis/issues/766)
+### [ocis-storage overwriting a file as share receiver, does not create a new file version for the sharer](https://github.com/owncloud/ocis/issues/766)
```
Comment (phil-davis) [inline tests/acceptance/expected-failures-API-on-OCIS-storage.md:2347]:
```suggestion
### [oCIS-storage overwriting a file as share receiver, does not create a new file version for the sharer](https://github.com/owncloud/ocis/issues/766)
```

File: tests/acceptance/expected-failures-API-on-OWNCLOUD-storage.md [hunk 4]
```diff
@@ -471,7 +471,7 @@ User cannot create a folder named Share
 
 ### [Split old public API webdav tests from new public webdav tests](https://github.com/owncloud/ocis-reva/issues/282)
 ### [Public link enforce permissions](https://github.com/owncloud/ocis/issues/1269)
-### [Previews via webDAV API tests fail on OCIS](https://github.com/owncloud/ocis/issues/187)
+### [Previews via webDAV API tests fail on CISS](https://github.com/owncloud/ocis/issues/187)
```
Comment (phil-davis) [inline tests/acceptance/expected-failures-API-on-OWNCLOUD-storage.md:474]:
```suggestion
### [Previews via webDAV API tests fail on oCIS](https://github.com/owncloud/ocis/issues/187)
```

I am not familiar with "CISS" - was that just an accident?

File: tests/acceptance/expected-failures-API-on-OWNCLOUD-storage.md [hunk 5]
```diff
@@ -1423,7 +1423,7 @@ User cannot create a folder named Share
 
 -   [apiSharePublicLink2/multilinkSharing.feature:181](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiSharePublicLink2/multilinkSharing.feature#L181)
 
-### [Previews via webDAV API tests fail on OCIS](https://github.com/owncloud/ocis/issues/187)
+### [Previews via webDAV API tests fail on CISS](https://github.com/owncloud/ocis/issues/187)
```
Comment (phil-davis) [inline tests/acceptance/expected-failures-API-on-OWNCLOUD-storage.md:1426]:
```suggestion
### [Previews via webDAV API tests fail on oCIS](https://github.com/owncloud/ocis/issues/187)
```

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
Thanks for getting this standardized!

**Comment by pascalwengerter** [comment]
> @pascalwengerter Thanks for getting this standardized!

Welcome, and thanks for the sharp eye during the review ;)

**Pull request merged**